lib/styles/modifiers/_shadows.scss

```diff
@@ -1,7 +0,0 @@
-.-shadowLarge {
```

Comment:
not used after https://github.com/bethinkpl/wnl-platform/pull/4966/commits/6f98ade1390f7c82ff5ec56f66dc09f278b22277